libturboparser/TurboParserInterface.cpp

```diff
@@ -12,304 +12,360 @@
 
 namespace TurboParserInterface {
 
-TurboTaggerWorker::TurboTaggerWorker() {
-  tagger_options_ = new TaggerOptions;
-  tagger_options_->Initialize();
+  TurboTaggerWorker::TurboTaggerWorker() {
```

Comment:
Namespaces should not be indented. See https://google.github.io/styleguide/cppguide.html#Namespaces.